src/components/shared/Button/Button.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import styles from './Button.module.css';
+import classNames from 'classnames/bind';
+
+const cx = classNames.bind(styles);
 
 export class Button extends React.Component {
     render() {
         return (
-            <button className={styles.button} data-cy={this.props.cy} onClick={this.props.handleClick} type={this.props.type || 'button'}>{this.props.text}</button>
+            <button className={cx(styles.button, {logout: this.props.cy === 'logout'})} data-cy={this.props.cy} onClick={this.props.handleClick} type={this.props.type || 'button'}>{this.props.text}</button>
```

Comment:
I think the Button class should remain general, and if you want to change positioning, can you wrap it in something wherever its being rendered and position that instead?